test/RibbonThetaVault.ts

```diff
@@ -227,6 +230,39 @@ describe("RibbonThetaVault", () => {
     availableChains: [CHAINID.ETH_MAINNET],
     protocol: OPTION_PROTOCOL.TD,
   });
+
+  behavesLikeRibbonOptionsVault({
+    name: `Ribbon RETH Theta Vault (Call)`,
+    tokenName: "Ribbon RETH Theta Vault",
+    tokenSymbol: "rRETH-THETA",
+    asset: RETH_ADDRESS[chainId],
+    assetContractName: "IWETH",
```

Comment:
Might be way the tests are failing, the contract abi is significantly different for RETH

Reply:
You already checked in the interface FYI just change it to `IRETH`
https://github.com/ribbon-finance/ribbon-v2/pull/255/files#diff-b75015944bb4beb6d69ef74161a50ad138c0f49378c122aef790b46940650106

Reply:
should be fine - theta vault doesnt call any reth specific methods (only erc20 standard methods)